mobile/src/modules/auth/store/actions.ts

```diff
@@ -231,6 +231,9 @@ export const setPasscode = createActionFn<string, Promise<void>>(
   async (dispatch, _getState, passcode) => {
     dispatch(actions.setPasscode(passcode));
     await savePasscode(passcode);
+
+    // reset the redux store
```

Comment:
hehe, if you need to explain the code, it's not comprehensive :stuck_out_tongue: Nevertheless, nice find, after so much struggle